src/Microsoft.ML.Recommender/RecommenderCatalog.cs

```diff
@@ -80,6 +80,8 @@ internal RecommendationTrainers(RecommendationCatalog catalog)
             /// <para>The basic idea of matrix factorization is finding two low-rank factor matrices to apporimate the training matrix.</para>
             /// <para>In this module, the expected training data is a list of tuples. Every tuple consists of a column index, a row index,
             /// and the value at the location specified by the two indexes. The training configuration is encoded in <see cref="MatrixFactorizationTrainer.Options"/>.
+            /// To invoke one-class matrix factorization, user needs to specify <see cref="MatrixFactorizationTrainer.LossFunctionType.SquareLossOneClass"/>.
+            /// The default settting <see cref="MatrixFactorizationTrainer.LossFunctionType.SquareLossRegression"/> is for standard matrix factorization problem.
```

Comment:
we're moving all the algo-details remarks to the estimator class. you can just link there instead of repeating the content.

Reply:
Maybe no because 1) this thing I have explained to an user 3 times on Gitter 2) this is the switch of implicit-feedback and explicit-feed recommendation --- a formula change rather than an algorithm change.

---
In reply to: [276869690](https://github.com/dotnet/machinelearning/pull/3409#discussion_r276869690) [](ancestors = 276869690)